<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewerWindow</class>
 <widget class="QMainWindow" name="viewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Cloud Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>651</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
    <widget class="QPushButton" name="btn_getCamera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Get position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_resetCamera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Point clouds</string>
    </property>
    <widget class="QTableWidget" name="table_cloud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>281</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>cloud</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_loadFile">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>320</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Open PCL file</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_cloud_color">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>380</y>
       <width>205</width>
       <height>31</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="columnCount">
      <number>8</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QLabel" name="label_cloud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinPointSize">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Size point</string>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
